vignettes/calibration.Rmd

```diff
@@ -108,13 +126,36 @@ The simulated trajectory is **not** sensible in this case - the `log_beta` value
 
 Setting $\log(\beta)=0$ instead gives us a trajectory that is still very unrealistic (the peak of our observed prevalence is only 16.7), but at least it's smooth. As it turns out this will be good enough, but finding appropriate starting values (based on external information and some trial and error) is often a significant part of a modeling workflow.
 
+---
+
+IP: Wait, why did simulator return a jagged line at all? Isn't the simulator deterministic?
+
+---
+
 ```{r plot2}
 (sir_simulator$report(c(0,0)) |>
  filter(matrix == "state", row == "I") |>
  ggplot(aes(time, value)) + geom_line()
 )
 ```
 
+---
+
+IP: What is the argument of `report()` above specifying?
+
+IP: This is a bit of a larger point that Steve may want to weigh in on, but the object-oriented approach here makes it a little more difficult to understand how to get help with various functions. For instance, I can't just do `?report` to get info on the arguments for the function above. Perhaps including a short section on the home page of `macpan2` quickly explaining that `macpan2` is object-oriented and how one can get help on object methods would be helpful. I know I had [something about this in the first Quickstart guide](https://canmod.github.io/macpan2/articles/quickstart.html#side-note-macpan2-is-object-oriented), but it didn't go as far as to explain how to get help, which I think is as simple as
+
+```
+> class(sir_simulator)
```

Comment:
Yes I completely agree @papsti .  Something like this in the vignette makes it even more important to make sure that the method documentation is up-to-date.  It also makes me want to write a `macpan_help` function that does basically what you just did.

Reply:
Yes that's a better idea.